wolfcrypt/src/port/caam/caam_qnx.c

```diff
@@ -1190,7 +1190,7 @@ static int getSupported(char* in)
 #endif
 
 char cannedResponse[] = {
-        "wolfCrypt QNX CAAM driver\n"
+        "wolfCrypt QNX CAAM driver version 4.7.1 (9)\n"
```

Comment:
Do we want to use LIBWOLFSSL_VERSION_STRING?
"wolfCrypt QNX CAAM driver version " LIBWOLFSSL_VERSION_STRING "\n"

Reply:
Avoided the LIBWOLFSSL_VERSION_STRING here because the driver should be detatched from main wolfcrypt/wolfssl code and not include any header files other than the caam_* files.